Utils/Editor/EditorGUIHelpers/OptionSelector.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using UnityEditor;
+
+namespace DUCK.Utils.Editor.EditorGUIHelpers
+{
+	public static partial class EditorGUILayoutHelpers
+	{
+		public static string OptionSelectorField(string label, string value, string[] options)
```

Comment:
At the moment if you pass in a value not found in the options it defaults to empty string, I guess we could provide a default value for such cases,